up42/auth.py

```diff
@@ -315,6 +320,7 @@ def _request(
 
         except requests.exceptions.RequestException as err:  # Base error class
```

Comment:
Let's change this type to requests.exceptions.HttpError

Reply:
after above 2 changes, following tests start failing with:
`requests.exceptions.HTTPError: {'data': {}, 'error': {'code': 429, 'message': 'rate limited'}}`

`tests/test_auth.py:208 test_request_rate_limited_retry`
`tests/test_auth.py:228 test_request_token_timeout_during_rate_limitation`